src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.cs

```diff
@@ -38,6 +34,7 @@ internal class NegoState
 
         private int _nestedAuth;
 
+        internal const int ERROR_TRUST_FAILURE = 1790;   // Used to serialize protectionLevel or impersonationLevel mismatch error to the remote side.
```

Comment:
Nit: ErrorTrustFailure?  Or does this naming exactly match some corresponding native definition?